### PR TITLE
zh: document handle backup gc behavior and how to delete backup

### PR DESCRIPTION
**Pull request by lichunzhu**
### What is changed, added or deleted? (Required)
Fix https://github.com/pingcap/docs-tidb-operator/issues/495 and fix https://github.com/pingcap/docs-tidb-operator/issues/376
document the gc behavior in backup and how to delete backup
document for PR https://github.com/pingcap/tidb-operator/pull/2897

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/backup-to-aws-s3-using-br.md [hunk 1]
```diff
@@ -516,3 +522,19 @@ kubectl get bk -l tidb.pingcap.com/backup-schedule=demo1-backup-schedule-s3 -n t
...
 + `.spec.schedule`：Cron 的时间调度格式。具体格式可参考 [Cron](https://en.wikipedia.org/wiki/Cron)。
 + `.spec.pause`：该值默认为 `false`。如果将该值设置为 `true`，表示暂停定时调度。此时即使到了调度时间点，也不会进行备份。在定时备份暂停期间，备份 Garbage Collection (GC) 仍然正常进行。将 `true` 改为 `false` 则重新开启定时全量备份。
+
+## 删除备份的 backup CR
+
+用户可以通过 `kubectl delete backup ${name} -n ${namespace}` 或 `kubectl delete backupschedule ${name} -n ${namespace}` 来删除对应的全量备份 CR 或定时全量备份 CR。
+
+tidb-operator v1.1.2 及以前版本或者在 v1.1.3 后版本将 `spec.cleanData` 设置为 true 时，operator 在删除 CR 时会同时删除备份出的文件。
+
+如果删除备份出的文件之前用户已手动将其删除，v1.1.2 版之前删除时可能会卡在 `Terminating` 状态。这时需要通过以前指令编辑对应的 backup/backupschedule CR 配置：
```
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:532]:
```suggestion
如果删除备份文件之前用户已手动将其删除，v1.1.2 及之前版本删除 CR 时会卡在 `Terminating` 状态。这时需要通过以下命令编辑对应的 backup/backupschedule CR 手动移除 `finalizer`：
```

**Comment by DanielZhangQD** [comment]
https://github.com/pingcap/tidb-operator/issues/2888#issuecomment-655836583 This is not covered.

**Comment by DanielZhangQD** [comment]
https://github.com/pingcap/tidb-operator/pull/2897 Please also add a description for this PR.

**Comment by lichunzhu** [comment]
@DanielZhangQD all comments have been addressed, PTAL again

File: zh/backup-to-aws-s3-using-br.md [hunk 2]
```diff
@@ -516,3 +524,23 @@ kubectl get bk -l tidb.pingcap.com/backup-schedule=demo1-backup-schedule-s3 -n t
...
+
+## 删除备份的 backup CR
+
+用户可以通过 `kubectl delete backup ${name} -n ${namespace}` 或 `kubectl delete backupschedule ${name} -n ${namespace}` 来删除对应的全量备份 CR 或定时全量备份 CR。
+
+tidb-operator v1.1.2 及以前版本或者在 v1.1.3 及以后版本将 `spec.cleanData` 设置为 true 时，TiDB Operator 在删除 CR 时会同时删除备份文件。
+
+建议用户在满足上述条件时如果需要删除 namespace，则首先删除所有的 backup/backupSchedule CR，再删除 namespace。
+
+如果直接删除存在 backup/backupSchedule 的 namespace，backup/backupSchedule 会持续尝试创建 finalizer 清理备份的数据，但因为 namespace 处于 `Terminating` 状态而创建失败，从而导致 namespace 卡在该状态。
```
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:536]:
```suggestion
如果直接删除存在 backup/backupSchedule 的 namespace，backup/backupSchedule 会持续尝试创建 Job 清理备份的数据，但因为 namespace 处于 `Terminating` 状态而创建失败，从而导致 namespace 卡在该状态。
```

File: zh/backup-to-aws-s3-using-br.md [hunk 3]
```diff
@@ -317,6 +317,20 @@ Amazon S3 支持以下几种 `storageClass` 类型：
 更多 `Backup` CR 字段的详细解释:
 
 * `.spec.metadata.namespace`：`Backup` CR 所在的 namespace。
+* `.spec.tikvGCLifeTime`：备份中的临时 `tikv_gc_lifetime` 时间设置，默认为 72h。
+
+    在备份开始之前，若 TiDB 集群的 `tikv_gc_lifetime` 小于用户设置的 `spec.tikvGCLifeTime`，为了保证备份的 tso 的数据不被 TiKV GC 掉，TiDB Operator 会在备份前[调节 `tikv_gc_lifetime`](https://pingcap.com/docs-cn/stable/dumpling-overview/#%E5%AF%BC%E5%87%BA%E5%A4%A7%E8%A7%84%E6%A8%A1%E6%95%B0%E6%8D%AE%E6%97%B6%E7%9A%84-tidb-gc-%E8%AE%BE%E7%BD%AE)为 `spec.tikvGCLifeTime`，并在备份成功后将 `tikv_gc_lifetime` 设置回原来的值。
```
Comment (ran-huang) [inline zh/backup-to-aws-s3-using-br.md:322]:
```suggestion
    在备份开始之前，若 TiDB 集群的 `tikv_gc_lifetime` 小于用户设置的 `spec.tikvGCLifeTime`，为了保证备份的 tso 的数据不被 TiKV GC 掉，TiDB Operator 会在备份前[调节 `tikv_gc_lifetime`](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview#%E5%AF%BC%E5%87%BA%E5%A4%A7%E8%A7%84%E6%A8%A1%E6%95%B0%E6%8D%AE%E6%97%B6%E7%9A%84-tidb-gc-%E8%AE%BE%E7%BD%AE)为 `spec.tikvGCLifeTime`，并在备份成功后将 `tikv_gc_lifetime` 设置回原来的值。
```

File: zh/backup-to-aws-s3-using-br.md [hunk 4]
```diff
@@ -317,6 +317,20 @@ Amazon S3 支持以下几种 `storageClass` 类型：
...
+
+    备份结束后不论成功或者失败，只要老的 `tikv_gc_lifetime` 比设置的备份 `.spec.tikvGCLifeTime` 小，TiDB Operator 都会尝试恢复 `tikv_gc_lifetime` 为备份前的值。在极端情况下，TiDB Operator 访问数据库失败会导致 TiDB Operator 无法自动恢复 `tikv_gc_lifetime` 并认为 backup 失败。
+
+    此时，可以通过下述语句查看当前 TiDB 集群的 `tikv_gc_lifetime`：
+
+    ```sql
+    select VARIABLE_NAME, VARIABLE_VALUE from mysql.tidb where VARIABLE_NAME like "tikv_gc_life_time";
+    ```
+
+    如果发现 `tikv_gc_lifetime` 值发现过大（通常为 10m），则需要按照[调节 `tikv_gc_lifetime`](https://pingcap.com/docs-cn/stable/dumpling-overview/#%E5%AF%BC%E5%87%BA%E5%A4%A7%E8%A7%84%E6%A8%A1%E6%95%B0%E6%8D%AE%E6%97%B6%E7%9A%84-tidb-gc-%E8%AE%BE%E7%BD%AE)将 `tikv_gc_lifetime` 调回原样。
```
Comment (ran-huang) [inline zh/backup-to-aws-s3-using-br.md:332]: ditto

File: zh/backup-to-aws-s3-using-br.md [hunk 5]
```diff
@@ -516,3 +530,23 @@ kubectl get bk -l tidb.pingcap.com/backup-schedule=demo1-backup-schedule-s3 -n t
 + `.spec.maxReservedTime`：一种备份保留策略，按时间保留备份。例如将该参数设置为 `24h`，表示只保留最近 24 小时内的备份条目。超过这个时间的备份都会被清除。时间设置格式参考 [`func ParseDuration`](https://golang.org/pkg/time/#ParseDuration)。如果同时设置最大备份保留个数和最长备份保留时间，则以最长备份保留时间为准。
 + `.spec.schedule`：Cron 的时间调度格式。具体格式可参考 [Cron](https://en.wikipedia.org/wiki/Cron)。
 + `.spec.pause`：该值默认为 `false`。如果将该值设置为 `true`，表示暂停定时调度。此时即使到了调度时间点，也不会进行备份。在定时备份暂停期间，备份 Garbage Collection (GC) 仍然正常进行。将 `true` 改为 `false` 则重新开启定时全量备份。
+
+## 删除备份的 backup CR
+
+用户可以通过 `kubectl delete backup ${name} -n ${namespace}` 或 `kubectl delete backupschedule ${name} -n ${namespace}` 来删除对应的全量备份 CR 或定时全量备份 CR。
```
Comment (ran-huang) [inline zh/backup-to-aws-s3-using-br.md:536]: 比较长的命令建议单独两行来放，加上 {{copyable}} 方便copy。

File: zh/backup-to-aws-s3-using-br.md [hunk 6]
```diff
@@ -516,3 +530,23 @@ kubectl get bk -l tidb.pingcap.com/backup-schedule=demo1-backup-schedule-s3 -n t
...
+
+tidb-operator v1.1.2 及以前版本或者在 v1.1.3 及以后版本将 `spec.cleanData` 设置为 true 时，TiDB Operator 在删除 CR 时会同时删除备份文件。
```
Comment (ran-huang) [inline zh/backup-to-aws-s3-using-br.md:538]:
```suggestion
如果你使用 v1.1.2 及以前版本，或使用 v1.1.3 及以后版本并将 `spec.cleanData` 设置为 true 时，TiDB Operator 在删除 CR 时会同时删除备份文件。
```
Is this what you mean?

File: zh/backup-to-aws-s3-using-br.md [hunk 7]
```diff
@@ -516,3 +530,23 @@ kubectl get bk -l tidb.pingcap.com/backup-schedule=demo1-backup-schedule-s3 -n t
...
+
+建议用户在满足上述条件时如果需要删除 namespace，则首先删除所有的 backup/backupSchedule CR，再删除 namespace。
```
Comment (ran-huang) [inline zh/backup-to-aws-s3-using-br.md:540]:
```suggestion
在满足上述条件时，如果需要删除 namespace，建议首先删除所有的 backup/backupSchedule CR，再删除 namespace。
```

File: zh/backup-to-aws-s3-using-br.md [hunk 8]
```diff
@@ -516,3 +530,23 @@ kubectl get bk -l tidb.pingcap.com/backup-schedule=demo1-backup-schedule-s3 -n t
...
+
+这时需要通过下述命令删除 `finalizer`：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl edit backup ${name} -n ${namespace}
+```
+
+在配置项项中删去所有的 `finalizer` 项，再次检查即可发现正常删除。
```
Comment (ran-huang) [inline zh/backup-to-aws-s3-using-br.md:552]:
```suggestion
在配置项中删去所有的 `finalizer` 项，再次检查即可发现正常删除。
```

File: zh/backup-to-gcs-using-br.md [hunk 9]
```diff
@@ -141,6 +141,20 @@ Ad-hoc 全量备份通过创建一个自定义的 `Backup` custom resource (CR)
 更多 `Backup` CR 字段的详细解释：
 
 * `.spec.metadata.namespace`：`Backup` CR 所在的 namespace。
+* `.spec.tikvGCLifeTime`：备份中的临时 `tikv_gc_lifetime` 时间设置，默认为 72h。
+
+    在备份开始之前，若 TiDB 集群的 `tikv_gc_lifetime` 小于用户设置的 `spec.tikvGCLifeTime`，为了保证备份的 tso 的数据不被 TiKV GC 掉，TiDB Operator 会在备份前[调节 `tikv_gc_lifetime`](https://pingcap.com/docs-cn/stable/dumpling-overview/#%E5%AF%BC%E5%87%BA%E5%A4%A7%E8%A7%84%E6%A8%A1%E6%95%B0%E6%8D%AE%E6%97%B6%E7%9A%84-tidb-gc-%E8%AE%BE%E7%BD%AE)为 `spec.tikvGCLifeTime`，并在备份成功后将 `tikv_gc_lifetime` 设置回原来的值。
```
Comment (ran-huang) [inline zh/backup-to-gcs-using-br.md:146]: comments 同 `zh/backup-to-aws-s3-using-br.md`

File: zh/backup-to-aws-s3-using-br.md (hunk 3 above)
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:322]:
```suggestion
    在备份开始之前，若 TiDB 集群的 `tikv_gc_lifetime` 小于用户设置的 `spec.tikvGCLifeTime`，为了保证备份的数据不被 TiKV GC 掉，TiDB Operator 会在备份前[调节 `tikv_gc_lifetime`](https://pingcap.com/docs-cn/stable/dumpling-overview/#%E5%AF%BC%E5%87%BA%E5%A4%A7%E8%A7%84%E6%A8%A1%E6%95%B0%E6%8D%AE%E6%97%B6%E7%9A%84-tidb-gc-%E8%AE%BE%E7%BD%AE) 为 `spec.tikvGCLifeTime`。
```

File: zh/backup-to-aws-s3-using-br.md [hunk 10]
```diff
@@ -317,6 +317,20 @@ Amazon S3 支持以下几种 `storageClass` 类型：
 更多 `Backup` CR 字段的详细解释:
 
 * `.spec.metadata.namespace`：`Backup` CR 所在的 namespace。
+* `.spec.tikvGCLifeTime`：备份中的临时 `tikv_gc_lifetime` 时间设置，默认为 72h。
+
+    在备份开始之前，若 TiDB 集群的 `tikv_gc_lifetime` 小于用户设置的 `spec.tikvGCLifeTime`，为了保证备份的 tso 的数据不被 TiKV GC 掉，TiDB Operator 会在备份前[调节 `tikv_gc_lifetime`](https://pingcap.com/docs-cn/stable/dumpling-overview/#%E5%AF%BC%E5%87%BA%E5%A4%A7%E8%A7%84%E6%A8%A1%E6%95%B0%E6%8D%AE%E6%97%B6%E7%9A%84-tidb-gc-%E8%AE%BE%E7%BD%AE)为 `spec.tikvGCLifeTime`，并在备份成功后将 `tikv_gc_lifetime` 设置回原来的值。
+
+    备份结束后不论成功或者失败，只要老的 `tikv_gc_lifetime` 比设置的备份 `.spec.tikvGCLifeTime` 小，TiDB Operator 都会尝试恢复 `tikv_gc_lifetime` 为备份前的值。在极端情况下，TiDB Operator 访问数据库失败会导致 TiDB Operator 无法自动恢复 `tikv_gc_lifetime` 并认为 backup 失败。
```
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:324]:
```suggestion
    备份结束后不论成功或者失败，只要老的 `tikv_gc_lifetime` 比设置的 `.spec.tikvGCLifeTime` 小，TiDB Operator 都会尝试恢复 `tikv_gc_lifetime` 为备份前的值。在极端情况下，TiDB Operator 访问数据库失败会导致 TiDB Operator 无法自动恢复 `tikv_gc_lifetime` 并认为备份失败。
```

File: zh/backup-to-aws-s3-using-br.md (hunk 4 above)
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:332]:
```suggestion
    如果发现 `tikv_gc_lifetime` 值发现过大（通常为 10m），则需要按照[调节 `tikv_gc_lifetime`](https://pingcap.com/docs-cn/stable/dumpling-overview/#%E5%AF%BC%E5%87%BA%E5%A4%A7%E8%A7%84%E6%A8%A1%E6%95%B0%E6%8D%AE%E6%97%B6%E7%9A%84-tidb-gc-%E8%AE%BE%E7%BD%AE) 将 `tikv_gc_lifetime` 调回原样。
```

File: zh/backup-to-aws-s3-using-br.md (hunk 7 above)
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:540]:
```suggestion
建议用户在满足上述条件时如果需要删除 namespace，则首先删除所有的 Backup/BackupSchedule CR，再删除 namespace。
```

File: zh/backup-to-aws-s3-using-br.md [hunk 11]
```diff
@@ -516,3 +530,23 @@ kubectl get bk -l tidb.pingcap.com/backup-schedule=demo1-backup-schedule-s3 -n t
...
+
+## 删除备份的 backup CR
+
+用户可以通过 `kubectl delete backup ${name} -n ${namespace}` 或 `kubectl delete backupschedule ${name} -n ${namespace}` 来删除对应的全量备份 CR 或定时全量备份 CR。
+
+tidb-operator v1.1.2 及以前版本或者在 v1.1.3 及以后版本将 `spec.cleanData` 设置为 true 时，TiDB Operator 在删除 CR 时会同时删除备份文件。
+
+建议用户在满足上述条件时如果需要删除 namespace，则首先删除所有的 backup/backupSchedule CR，再删除 namespace。
+
+如果直接删除存在 backup/backupSchedule 的 namespace，backup/backupSchedule 会持续尝试创建 Job 清理备份的数据，但因为 namespace 处于 `Terminating` 状态而创建失败，从而导致 namespace 卡在该状态。
```
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:542]:
```suggestion
如果直接删除存在 Backup/BackupSchedule CR 的 namespace，TiDB Operator 会持续尝试创建 Job 清理备份的数据，但因为 namespace 处于 `Terminating` 状态而创建失败，从而导致 namespace 卡在该状态。
```

File: zh/backup-to-aws-s3-using-br.md (hunk 8 above)
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:552]:
```suggestion
删除 `metadata.finalizers` 配置，即可正常删除 CR。
```

File: zh/backup-to-aws-s3-using-br.md [hunk 12]
```diff
@@ -516,3 +530,23 @@ kubectl get bk -l tidb.pingcap.com/backup-schedule=demo1-backup-schedule-s3 -n t
...
+
+这时需要通过下述命令删除 `finalizer`：
```
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:544]:
```suggestion
这时需要通过下述命令删除 `finalizers`：
```

**Comment by lichunzhu** [comment]
@ran-huang @DanielZhangQD All comments have been addressed. PTAL again

File: zh/backup-to-aws-s3-using-br.md [hunk 13]
```diff
@@ -310,13 +310,27 @@ Amazon S3 支持以下几种 `storageClass` 类型：
...
+
+    备份结束后不论成功或者失败，只要老的 `tikv_gc_lifetime` 比设置的 `.spec.tikvGCLifeTime` 小，TiDB Operator 都会尝试恢复 `tikv_gc_lifetime` 为备份前的值。在极端情况下，TiDB Operator 访问数据库失败会导致 TiDB Operator 无法自动恢复 `tikv_gc_lifetime` 并认为备份失败。
+
+    此时，可以通过下述语句查看当前 TiDB 集群的 `tikv_gc_lifetime`：
+
+    ```sql
+    select VARIABLE_NAME, VARIABLE_VALUE from mysql.tidb where VARIABLE_NAME like "tikv_gc_life_time";
+    ```
+
+    如果发现 `tikv_gc_lifetime` 值发现过大（通常为 10m），则需要按照[调节 `tikv_gc_lifetime`](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview#导出大规模数据时的-tidb-gc-设置) 将 `tikv_gc_lifetime` 调回原样。
```
Comment (ran-huang) [inline zh/backup-to-aws-s3-using-br.md:332]:
```suggestion
    如果发现 `tikv_gc_lifetime` 值过大（通常为 10m），则需要按照[调节 `tikv_gc_lifetime`](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview#导出大规模数据时的-tidb-gc-设置) 将 `tikv_gc_lifetime` 调回原样。
```

File: zh/backup-to-gcs-using-br.md [hunk 14]
```diff
@@ -141,6 +141,20 @@ Ad-hoc 全量备份通过创建一个自定义的 `Backup` custom resource (CR)
...
+
+    备份结束后不论成功或者失败，只要老的 `tikv_gc_lifetime` 比设置的 `.spec.tikvGCLifeTime` 小，TiDB Operator 都会尝试恢复 `tikv_gc_lifetime` 为备份前的值。在极端情况下，TiDB Operator 访问数据库失败会导致 TiDB Operator 无法自动恢复 `tikv_gc_lifetime` 并认为备份失败。
+
+    此时，可以通过下述语句查看当前 TiDB 集群的 `tikv_gc_lifetime`：
+
+    ```sql
+    select VARIABLE_NAME, VARIABLE_VALUE from mysql.tidb where VARIABLE_NAME like "tikv_gc_life_time";
+    ```
+
+    如果发现 `tikv_gc_lifetime` 值发现过大（通常为 10m），则需要按照[调节 `tikv_gc_lifetime`](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview#导出大规模数据时的-tidb-gc-设置) 将 `tikv_gc_lifetime` 调回原样。
```
Comment (ran-huang) [inline zh/backup-to-gcs-using-br.md:156]:
```suggestion
    如果发现 `tikv_gc_lifetime` 值过大（通常为 10m），则需要按照[调节 `tikv_gc_lifetime`](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview#导出大规模数据时的-tidb-gc-设置) 将 `tikv_gc_lifetime` 调回原样。
```

File: zh/backup-to-s3.md [hunk 15]
```diff
@@ -242,6 +242,20 @@ Amazon S3 支持以下几种 `storageClass` 类型：
...
+
+    备份结束后不论成功或者失败，只要老的 `tikv_gc_lifetime` 比设置的 `.spec.tikvGCLifeTime` 小，TiDB Operator 都会尝试恢复 `tikv_gc_lifetime` 为备份前的值。在极端情况下，TiDB Operator 访问数据库失败会导致 TiDB Operator 无法自动恢复 `tikv_gc_lifetime` 并认为备份失败。
+
+    此时，可以通过下述语句查看当前 TiDB 集群的 `tikv_gc_lifetime`：
+
+    ```sql
+    select VARIABLE_NAME, VARIABLE_VALUE from mysql.tidb where VARIABLE_NAME like "tikv_gc_life_time";
+    ```
+
+    如果发现 `tikv_gc_lifetime` 值发现过大（通常为 10m），则需要按照[调节 `tikv_gc_lifetime`](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview#导出大规模数据时的-tidb-gc-设置) 将 `tikv_gc_lifetime` 调回原样。
```
Comment (ran-huang) [inline zh/backup-to-s3.md:257]:
```suggestion
    如果发现 `tikv_gc_lifetime` 值过大（通常为 10m），则需要按照[调节 `tikv_gc_lifetime`](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview#导出大规模数据时的-tidb-gc-设置) 将 `tikv_gc_lifetime` 调回原样。
```

File: zh/backup-to-gcs.md [hunk 16]
```diff
@@ -245,3 +259,34 @@ kubectl get bks -n test1 -owide
...
+kubectl delete backupschedule ${name} -n ${namespace}
+```
+
+如果你使用 v1.1.2 及以前版本，或使用 v1.1.3 及以后版本并将 `spec.cleanData` 设置为 true 时，TiDB Operator 在删除 CR 时会同时删除备份文件。
+
+在满足上述条件时，如果需要删除 namespace，建议首先删除所有的 Backup/BackupSchedule CR，再删除 namespace。
+
+如果直接删除存在 Backup/BackupSchedule CR 的 namespace，TiDB Operator 会持续尝试创建 Job 清理备份的数据，但因为 namespace 处于 `Terminating` 状态而创建失败，从而导致 namespace 卡在该状态。
+
+同时 v1.1.2 及以前版本如果删除 Backup/BackupSchedule CR 时备份文件已被手动清除，也会出现卡 `Terminating` 的情况。
```
Comment (ran-huang) [inline zh/backup-to-gcs.md:282]:
```suggestion
同时 v1.1.2 及以前版本如果删除 Backup/BackupSchedule CR 时备份文件已被手动清除，也会出现卡在 `Terminating` 的情况。
```

File: zh/backup-to-s3.md [hunk 17]
```diff
@@ -521,3 +535,34 @@ kubectl get bk -l tidb.pingcap.com/backup-schedule=demo1-backup-schedule-s3 -n t
...
+kubectl delete backupschedule ${name} -n ${namespace}
+```
+
+如果你使用 v1.1.2 及以前版本，或使用 v1.1.3 及以后版本并将 `spec.cleanData` 设置为 true 时，TiDB Operator 在删除 CR 时会同时删除备份文件。
+
+在满足上述条件时，如果需要删除 namespace，建议首先删除所有的 Backup/BackupSchedule CR，再删除 namespace。
+
+如果直接删除存在 Backup/BackupSchedule CR 的 namespace，TiDB Operator 会持续尝试创建 Job 清理备份的数据，但因为 namespace 处于 `Terminating` 状态而创建失败，从而导致 namespace 卡在该状态。
+
+同时 v1.1.2 及以前版本如果删除 Backup/BackupSchedule CR 时备份文件已被手动清除，也会出现卡 `Terminating` 的情况。
```
Comment (ran-huang) [inline zh/backup-to-s3.md:558]:
```suggestion
同时 v1.1.2 及以前版本如果删除 Backup/BackupSchedule CR 时备份文件已被手动清除，也会出现卡在 `Terminating` 的情况。
```

**Pull request merged**